index.html

```diff
@@ -1,11 +1,18 @@
-<<!DOCTYPE html>
+<!DOCTYPE html>
 <html>
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title>Boncingballs</title>
+	<title>Bouncingballs</title>
+	<script src="index.js"></script>
+	<link rel="stylesheet" href="styles/index.css">
+	<link rel="reset" href="styles/reset.css">
 </head>
-<body>
 
+<body>
+	<canvas 
+		id="myCanvas" 
```

Comment:
nit: not sure new lines are needed for such a short tag

Reply:
good catch, the new code had not pushed which was why ther were not in one line.